src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Random must be a class variable
```suggestion
        int index = random.nextInt(Color.values().length);
```